### PR TITLE
documentation updates

### PR DESCRIPTION
**Pull request by JeffRiggle**
Just wanted to give some feedback on the README.md. These are all things I kind of stumbled through on the first little bit of working with this. The biggest issue I had was my node version was 12.4.x. I figure this would just be generally useful

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,40 @@ function getCustomClientData() {
...
+##### On Mac
+It would be recommended to manage node version using something like nvm
+```
+brew install nvm
+nvm use 10.15.3
+npm ci
+```
+
+##### On Windows or Linux
+Make sure the correct version of node is being used and run install
```
Comment (covertcj) [inline README.md]: What's the 'correct' version of node?  Was 12.4 just too old?

Comment (covertcj) [reply README.md]: Oh, nevermind, I can see that above... Why is 10.15.3 necessary?

Comment (JeffRiggle) [reply README.md]: 10.15.3. 12.4 was too new. I think this had something to do with our dependencies something like a compile of node-gyp failed. I imagine we will eventually want to look into that as well

Comment (MattCheely) [reply README.md]: I would just note that consumers should build with the latest node LTS version and leave it at that. We should be making sure it works consistently as new releases come out, and we won't need to update the docs every time there's a new release.

I'd also leave out specific brew/nvm steps for setting up a node version. We can tell folks what version they should use, but there's no real reason for us to maintain instruction on how to manage node.

**Pull request merged**